main.py

```diff
@@ -1,10 +1,74 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as players_json:
+        players = json.load(players_json)
+        for nickname, bio in players.items():
+            guild = get_or_create_guild(bio)
+            race = get_or_create_race(bio)
+            create_skills(bio, race)
+            create_player(nickname=nickname, race=race, guild=guild, bio=bio)
+
+
+def create_skills(bio: dict, race: Race) -> None:
+    skills_list = bio.get("race").get("skills")
```

Comment:
Consider checking if the 'skills' key exists in the 'bio' dictionary before attempting to access it. This will prevent potential KeyError exceptions if the 'skills' key is missing.